CHANGELOG.md

```diff
@@ -12,6 +12,19 @@ follows:
    releases, and Linux kernel updates. They're also made to fix bugs and add
    features to the build infrastructure.
 
+## v1.22.2
+
+This is a bug fix and Erlang version bump from 25.2 to 25.2.3. It should be a
+low risk upgrade from v1.22.0.
```

Comment:
```suggestion
This is a bug fix and Erlang version bump from 25.2.3 to 25.3. It should be a
low risk upgrade from v1.22.1.
```

Reply:
Shoot. I totally missed that copy/paste. Thank you!